docs/whats_new.rst

```diff
@@ -30,6 +30,7 @@ Enhancements
 - Adding Mixup augmentation :class:`braindecode.augmentation.Mixup` (:gh:`254` by `Simon Brandt`_)
 - Adding saving of preprocessing and windowing choices in :func:`braindecode.preprocessing.preprocess`, :func:`braindecode.preprocessing.create_windows_from_events` and :func:`braindecode.preprocessing.create_fixed_length_windows` to datasets to facilitate reproducibility (:gh:`287` by `Lukas Gemein`_)
 - Adding :func:`braindecode.models.util.aggregate_probas` to perform self-ensembling of predictions with sequence-to-sequence models (:gh:`294` by `Hubert Banville`_)
+- Adding :func:`braindecode.training.scoring.predict_trials` to generate trialwise predictions (:gh:`312` by `Lukas Gemein`_)
```

Comment:
```suggestion
- Adding :func:`braindecode.training.scoring.predict_trials` to generate trialwise predictions after cropped training (:gh:`312` by `Lukas Gemein`_)
```

To make even clearer

Reply:
I sitll disagree. For me this is not specific for cropped decoding. See comment above.

Reply:
@robintibor In `braindecode.training.scoring.predict_trials` we cannot know whether the model was trained in cropped fashion. We need an `EEGClassifier` / `EEGRegressor` for this. Is it save to add this function anyways? Or should we remove it and only have `EEGClassifier/EEGRegressor.predict_trials()` which will then check `self.cropped`?

Reply:
What about this here @robintibor

Reply:
I think we should have it so people can use this if they are not using skorch.